parsers/CR.py

```diff
@@ -202,62 +175,65 @@ def fetch_production(
 
     # Do not use existing session as some amount of cache is taking place
     r = Session()
-    url = "https://apps.grupoice.com/CenceWeb/CencePosdespachoNacional.jsf"
-    response = r.get(url)
-
-    soup = BeautifulSoup(response.text, "html.parser")
-    jsf_view_state = soup.find("input", {"name": "javax.faces.ViewState"})["value"]
-
-    data = [
-        ("formPosdespacho:txtFechaInicio_input", target_datetime.format(DATE_FORMAT)),
-        ("formPosdespacho:pickFecha", ""),
-        ("formPosdespacho_SUBMIT", 1),
-        ("javax.faces.ViewState", jsf_view_state),
-    ]
-    response = r.post(url, data=data)
-
-    # tell pandas which table to use by providing CHARACTERISTIC_NAME
-    df = pd.read_html(
-        response.text, match=CHARACTERISTIC_NAME, skiprows=1, index_col=0
-    )[0]
-
-    results = df_to_data(zone_key, target_datetime, df, logger)
+    day = target_datetime.format("DD")
+    month = target_datetime.format("MM")
+    year = target_datetime.format("YYYY")
```

Comment:
`target_datetime` have the `datetime` type which don't have a format function but is assigned a arrow object higher up. This causes multiple type errors.